articles/support/_delete-tenant.md

```diff
@@ -1,21 +1,21 @@
-## Delete your Auth0 Tenant
+## Delete tenants
+
+This process deletes only one tenant. If you have other tenants that need to be removed, you will need to repeat these steps for each tenant.
 
 ::: warning
-This action cannot be undone.
+You **cannot** restore a deleted tenant and you cannot reuse a tenant name when creating new tenants. 
 :::
 
-1. Log in to the [dashboard](${manage_url}).
-2. Click on your username in the top right corner to bring up the associated dropdown box.
+::: panel Enterprise customers
+If you are an Enterprise customer, you may have multiple child tenants as part of your Auth0 contract. Rest assured that if you choose to delete such a tenant, this will have no impact on your agreement with Auth0. We recommend that if you have any concerns about your Enterprise agreement, then discuss them with your Customer Success Manager (CSM).
```

Comment:
```suggestion
If you are an Enterprise customer, you may have multiple child tenants as part of your Auth0 contract. Rest assured that if you choose to delete such a tenant, this will have no impact on your agreement with Auth0. We recommend speaking with your Customer Success Manager (CSM) if you have any concerns about your Enterprise agreement.
```